frontend/src/libraries/explorer-wamp/details.js

```diff
@@ -24,7 +24,7 @@ export default class DetailsApi extends ExplorerApi {
               WHERE blocks.timestamp > (strftime('%s','now') - 10) * 1000
           ) as transactions_per_second,
           (SELECT height as lastBlockHeight FROM blocks ORDER BY height DESC LIMIT 1) as last_block,
-          (SELECT COUNT(DISTINCT signer_id) as accountsCount FROM transactions) as total_accounts`
+          (SELECT COUNT(*) as accountsCount FROM accounts) as total_accounts`
```

Comment:
this may not inclueded genesis aaccount, we should sum all accounts. Maybe we can left this here, but after getting genesis information, we should fix it

Reply:
Well, counting DISTINCT for all the transactions is computationally heavy, and still does not provide a good estimate on the total count of accounts, and on the accounts page we only display the accounts we have in the table anyway, so it is just more consistent. We should, however, sync genesis info... https://github.com/nearprotocol/near-explorer/issues/185